webrtc.html

```diff
@@ -3147,27 +3147,8 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
 
         </dl>
         <dl class="idl" title="dictionary RTCRtpTransceiverInit">
-          <dt>boolean send = true</dt>
-
-          <dd>
-            <p>If true, indicates that
-            the <a><code>RTCRtpTransceiver</code></a>'s <a><code>RTCRtpSender</code></a>
-            will offer to send RTP and send RTP if the remote peer accepts.  If false,
-            indicates that the <a><code>RTCRtpSender</code></a> will not offer to send RTP
-            and will not send RTP (the direction of the <a>media description</a> generated by createOffer
-            will be <code>recvonly</code> or <code>inactive</code>).</p>
-          </dd>
-
-          <dt>boolean receive = true</dt>
-
-          <dd>
-            <p>If true, indicates that
-            the <a><code>RTCRtpTransceiver</code>'</a>s <a><code>RTCRtpReceiver</code></a>
-            will offer to receive RTP and receive RTP if the remote peer accepts.  If
-            false, indicates that the <a><code>RTCRtpReceiver</code></a> will not offer to
-            receive RTP and will not receive RTP (the direction of the <a>media description</a> generated by
-            createOffer will be <code>sendonly</code> or <code>inactive</code>).</p>
-          </dd>
+          <dt>RTCRtpDirectionality directionality = "sendrecv"</dt>
+          <dd>The directionality of the RTP stream.</dd>
```

Comment:
Or better yet, add that "or boolean" convenience trick from PR 471 and use it here:
RTCRtpActivity send = "active",
RTCRtpActivity receive = "active"


Then {send: false} will still work, which would be nice.